ActindoWorkFlow/UserInterface/08_CoreActions.md

```diff
@@ -0,0 +1,247 @@
+
+
+
+## Wait for criterion
+
+![Wait for criterion](../../Assets/Screenshots/ActindoWorkFlow/Workflows/CoreActions/WaitForCriterion.png "[Wait for criterion]")
+
+The *Wait for criterion* core action is used to insert a breakpoint when you must wait that the data input matches a certain criterion.   
+The data runs via the *in* input port into the workflow action and is output via the *match* output port once the input value matches the criterion to be compared with. If the input value does not match the criterion to be compared with after a defined timeout, the data is, depending on the configuration, either output via the *timeout* output port, or the actions fails.   
+The criterion to be compared with as well as the wait time settings are defined in the configuration.
+
+> [Info] It is recommended to give the action a name that describes which input value the criterion is waiting for.
+
+### Configuration  
+
+- *Initial wait time*   
+  Enter the time in minutes to wait until the next comparison if the criterion does not initially match the input data.
+
+  > [Info] The initial wait time defines the time after the initial comparison, not until the initial comparison. The initial comparison is executed automatically when the action is triggered for the first time.
+
+- *Wait time interval*   
+  Enter the time in minutes to wait for each additional comparison attempt if the criterion does not yet match the input data.
+
+- *Timeout*   
+  Enter the time in minutes after which no further attempts should be started and the input data will either be output via the *timeout* output port or the action will fail.
+
+- ![Toggle](../../Assets/Icons/Toggle.png "[Toggle]") *Fail on timeout*   
+  Enable this toggle to make the action fail after the defined timeout. Disable the toggle to output the current input value via the *timeout* output port after the defined timeout. By default, this toggle is disabled.
+
+- *Path*   
+  Enter the path to the property of the input object to be compared with.
+
+  > [Info] The properties of an entity you can refer to are described in the relevant API documentation.
+
+
+- *Operator*   
+  Enter the applicable operator for the comparison. The following operators are available:
+  - **=**  
+    It is a match if the input value equals the defined value.
+  - **!=**    
+    It is a match if the input value does not equals the defined value.
+  - **in**    
+    It is a match if the input value is included in the defined value. For this operator, it is possible to enter an array in the *Value* field.
+  - **notIn**   
+    It is a match if the input value is not included in the defined value. For this operator, it is possible to enter an array in the *Value* field.
+  - **regexp**   
+    It is a match if the input value equals the regular expression entered in the *Value* field.    
+    For detailed information about regular expressions, see https://regex101.com/.
```

Comment:
Vorschlag (verwendet in Taxes): The site https://regex101.com/ can be useful to test regular expressions.